pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
-requires = ["setuptools", "wheel"]
```

Comment:
Setuptools' docs used to incorrectly say you should include `wheel`. They've fixed that.